explorer/src/queries/search.ts

```diff
@@ -15,12 +21,21 @@ const searchBuilder = (
 
   if (params.searchQuery != null) {
     const searchTokens = params.searchQuery.split(/\s+/)
+    const normalizedSearchTokens = searchTokens.map(token =>
```

Comment:
I think this can be reduced to `searchTokens(normalizeSearchToken)`

Reply:
Isn't searchTokens an array and normalizeSearchToken a function that acts on a single string?

Reply:
Sorry. That's my bad. I meant

```
searchTokens.map(normalizeSearchToken)
```